42/README.md

```diff
@@ -70,15 +71,144 @@ The above process ensure the node keeps validating and accepting new blocks whil
 
 
 ### Type 3 scenarios
-When Tendermint migrates to a new version wherein new validation rules are not compatible with the existing blockchain it induces a hard fork. Tendermint encourages its users to create a new blockchain and archive the current blockchain at particular agreed upon (by the network) height. In order to support such changes BigchainDB would also need to archive the existing blockchain because the alternative approach is to run two Tendermint processes, one with the older validation logic and another with the newer validation logic. Since the latter approach is a huge hassle for BigchainDB users it would be best to recommend our user to archive the blockchain.
 
-Following describes how to archive and start a new blockchain,
-- Create a TEP which proposes a block height `h` at which the current blockchain will stop creating any new blocks.
-- Once the TEP concludes and the height `h` has been chosen, validators stop proposing new blocks once height `h` has been reached.
-- At height `h` the merkle root of the UTXO of the current blockchain is provided as the genesis state of the new blockchain.
-- The validator set at height `h` should be used as the validator set in the genesis of the new blockchain.
-- Since from Tendermint's point of view this is a new blockchain, block height will start from `0` which implies that BigchainDB would need to archive the existing blocks collection. All other collections would remain as is.
+This section advises on how to approach backwards-incompatible upgrades of the Tendermint chain - the type of updgrades when Tendermint can not build blocks on top of the existing chain.
 
+BigchainDB operators need a convenient way to do the following:
+
+1. Stop building blocks when the old Tendermint chain is at a jointly chosen height.
```

Comment:
This line makes it seem like the operators have to agree on a migration height (e.g. 4592) ahead of time, but later on, you make it clear that the migration height is just whatever the current height happens to be when the election is concluded. Maybe change this line to read:

1. Stop building blocks for the old Tendermint chain at the same height as all the other operators.